.github/workflows/pages-deploy.yml.hook

```diff
@@ -1,29 +1,61 @@
-name: 'Automatic build'
+name: "Automatic build and deploy"
 on:
+  workflow_dispatch:
   push:
     branches:
-      - master
```

Comment:
The event branch be reverted to `master`

Reply:
Added `master` so it can trigger on either, in case folks want to use either name.